spec/consul_application_settings_spec.rb

```diff
@@ -11,66 +11,4 @@
     set_consul_value('foo', 'bar')
     expect(Diplomat::Kv.get('foo')).to eq('bar')
   end
-
-  describe '.configure' do
```

Comment:
This spec is general integration spec and also verification of the config. Instead of simply removing - maybe make sense to move configuration examples somewhere else?

Reply:
Fixed. Added test for `.configure`.